documents/api/viewsets.py

```diff
@@ -69,8 +69,9 @@ def get_queryset(self):
 
         return Attachment.objects.filter(**qs_filters)
 
-    def retrieve(self, request, *args, **kwargs):
-        raise MethodNotAllowed(request.method)
+    @staff_required(required_permission=ServicePermissions.VIEW_DOCUMENTS)
```

Comment:
Should we use `VIEW_ATTACHMENTS` permission here instead?

Reply:
Good catch!